### DISCUSSION TITLE
Clone Git repository in new system

### DISCUSSION THREAD

**Discussion by vijayavanisabbiti** [learndevopsonline/learndevopsonline#1646] [Q&A] [answered]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/150422689/b6be8c8a-4014-4184-b424-9109d67960d7)

Continuation of old thread: #1213 
[sreedharm07](https://github.com/sreedharm07)
[47 minutes ago](https://github.com/learndevopsonline/learndevopsonline/discussions/1213#discussioncomment-7700936)
are you using office laptop?

Yes, I'm using office laptop.

**Comment by sreedharm07** [comment, marked as answer]
@vijayavanisabbiti 
your office laptop may have some firewalls thats why its not clonning

**Reply by vijayavanisabbiti** [reply]
Okay, thank you.
